Bottom: 4ba810928c534b97666dff271e7fb3c3e7fa98c4
Top:    ddf5846a4356fb0b547b7f39c3290cd66247a036
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-08-28 02:16:26 +0100

Refresh of hid-gtk-remove-unused-colour-a

---

diff --git a/src/hid/gtk/gui-top-window.c b/src/hid/gtk/gui-top-window.c
index 9641a25..beef690 100644
--- a/src/hid/gtk/gui-top-window.c
+++ b/src/hid/gtk/gui-top-window.c
@@ -224,7 +224,7 @@ GhidGui _ghidgui, *ghidgui = NULL;
 
 GHidPort ghid_port, *gport;
 
-static GdkColor WhitePixel, BlackPixel;
+static GdkColor WhitePixel;
 
 static gchar		*bg_image_file;
 
@@ -2504,7 +2504,6 @@ ghid_create_pcb_widgets (void)
   GError	*err = NULL;
 
   gdk_color_parse ("white", &WhitePixel);
-  gdk_color_parse ("black", &BlackPixel);
 
   if (bg_image_file)
     ghidgui->bg_pixbuf = gdk_pixbuf_new_from_file(bg_image_file, &err);
